src/main.rs

```diff
@@ -26,11 +30,11 @@ fn main() {
 
 fn set_ctrl_c_handler() {
     ctrlc::set_handler(|| {
-        println!("received Ctrl+C!");
+        info!("Shutdown Uption");
```

Comment:
Maybe warning here would be better?
```suggestion
        warning!("Shutdown Uption");
```